subsys/net/l2/openthread/openthread.c

```diff
@@ -497,6 +506,8 @@ static int openthread_init(struct net_if *iface)
 		.no_yield = true,
 	};
 
+	otError err;
```

Comment:
Let's keep variable definitions together, I'm not sure why is there an empty line inbetween the first two.